examples/complete/main.tf

```
@@ -65,7 +68,8 @@ module "sql" {
 }
 
 module "database" {
-  # source = "github.com/equinor/terraform-azurerm-sql//modules/database?ref=v0.0.0"
+  # source  = "equinor/sql/azurerm//modules/database"
+  # version = "10.2.3"
```

I'd just remove the suggested version pin for local modules, since it'll be difficult to maintain.

```suggestion
  # source = "equinor/sql/azurerm//modules/database"
```

---

I'd remove it for all local module calls, so it's totally random that I'm pointing out this specific one 🙂

---

Allright, the thought behind it was for it to be a kinda log as to when the examples was last updated, but I see your point! I'll remove it 👍🏼

---

```suggestion
  # source  = "equinor/sql/azurerm//modules/database"
  # version = "0.0.0"
```

Could we something like this instead?